openlibrary/plugins/openlibrary/js/covers.js

```diff
@@ -112,3 +112,76 @@ export function initCoversSaved() {
         parent.$(cover_selector).attr('src', cover_url);
     }
 }
+
+// This function will be triggered when the user clicks the "Paste" button
+export async function pasteImage() {
+    let formData = null;
+    try {
+        const clipboardItems = await navigator.clipboard.read();
+        for (const item of clipboardItems) {
+            if (item.types.includes('image/png')) {
```

Comment:
Ah there's a bug here; since there can be multiple items in the clipboardItems, you'll want to _skip_ any non image types, not error. Eg

```js
if (!item.types.includes('image/png')) {
    continue;
}
```

And also check for jpg images as well, since those will work too!

Reply:
Yes done. Additionally, initially I had included only png support because clipboards store all copied image formats in .png irrespective of the initial image format. However after a little digging after your suggestion, I found out that in Windows, users can changes their settings to store images in the jpg format. I may have to dig deeper into this, but I do not see any harm in adding the support right now.